src/scanner.c

```diff
@@ -105,6 +105,66 @@ Obj             EndLineHook = 0;
 
 /****************************************************************************
 **
+*F  GET_CHAR()  . . . . . . . . . . . . . . . . get the next character, local
+**
+**  'GET_CHAR' returns the next character from  the current input file.  This
+**  character is afterwords also available as '*In'.
+**
+**  For efficiency  reasons 'GET_CHAR' is a  macro that  just  increments the
```

Comment:
`GET_CHAR` isn't a macro anymore. I'd just loose the first part of this paragraph and start maybe like this:
>  'GET_CHAR' increments the pointer ...